bazel/osx/BUILD

```diff
@@ -0,0 +1,46 @@
+licenses(["notice"])  # Apache 2
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"] + ["**/*"]),
+)
+
+exports_files([
+    "xcode_locator.m",
+])
+
+DARWIN_XCODE_LOCATOR_COMPILE_COMMAND = """
+  /usr/bin/xcrun clang -fobjc-arc -framework CoreServices \
+      -framework Foundation -o $@ $<
+"""
+
+genrule(
```

Comment:
I don't think @bazel_tools is defined by Bazel. The only way to do this would be to include the Bazel git tree as a `git_repository` and then reference these files. I think that would be reasonable to do for unchanged subtrees. Looking below, I think it might make sense to do this.